website/docs/r/log_analytics_workspace.html.markdown

```diff
@@ -39,6 +39,8 @@ The following arguments are supported:
 
 * `allow_resource_only_permissions` - (Optional) Specifies if the log Analytics Workspace allow users accessing to data associated with resources they have permission to view, without permission to workspace. Defaults to `true`.
 
+* `disable_local_auth` - (Optional) Specifices if the log Analytics workspace should enforce authentication using Azure AD. Defaults to `false`.
```

Comment:
as this is a boolean, it should be consistent with other booleans in the provider (enabled/disabled on the end of the name)
```suggestion
* `local_authentication_disabled` - (Optional) Specifices if the log Analytics workspace should enforce authentication using Azure AD. Defaults to `false`.
```